### ISSUE TITLE
`@apply` cannot be used with `.transparent`, `text-l` and probably a bunch of other classes - but works fine with others

### ISSUE THREAD

**Issue by leoncvlt** [tailwindcss/tailwindcss#1518]
Hi, I am using tailwinds css with webpack and postcss. My setup is:

webpack.config.js - module/rules:
```
        {
          test: /\.css$/,
          use: ["postcss-loader"],
        },
```

postcss.config.js:
```
const tailwindcss = require("tailwindcss");
const autoprefixer = require("autoprefixer");

module.exports = {
  plugins: [
    tailwindcss,
    autoprefixer,
  ]
};
```

main.css imported in the entry point:
```
@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";
```

Quite a basic setup - everything seems to work out of the box.

I tried using `@apply` to create som reusable components, and weirdly enough it seems to work for some classes, but not for others. If I add this to my main.css:

```
@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";

.my-button {
  @apply bg-blue-600 text-white;
}

.my-container {
  @apply transparent text-l w-20 flex flex-col items-center;
}
```

`my-button` applies fine, but then I get 
```
`@apply cannot be used with `.transparent` because `.transparent` either cannot be found, or its actual definition includes a pseudo-selector like :hover, :active, etc. If you're sure that `.transparent` exists, make sure that any `@import` statements are being properly processed *before* Tailwind CSS sees your CSS, as `@apply` can only be used for classes in the same CSS tree.`. 
```
If I remove `transparent` from `my-container`, it fails with `text-l`. If I remove both, it works fine.

Trying to get my head around this.

**Comment by adamwathan** [comment]
Tailwind doesn’t have a class called “transparent” or “text-l”, are those custom classes you added yourself?

**Comment by leoncvlt** [comment]
Welp - you are right. Those are not even classes I added, I just wrote them down assuming they were part of tailwind's set of utilities. For some reason in my mind I imagined that adding non-existing classes would generate some sort of warning of error (which, well, it did when using @apply).

Sorry to have wasted your time! Thanks a lot for tailwind, and congrats on the launch of tailwind UI. Closing this now!
